From 001673df31b20351f50cbd0939538b5c0fc050d7 Mon Sep 17 00:00:00 2001
From: Shivani Raval <sraval@habana.ai>
Date: Fri, 23 May 2025 13:46:31 +0300
Subject: [PATCH] [JSW-31] specs: added missing file in specs

Added file related to RMA for JGS preboot asic build.

Change-Id: Ie23dd299a1ab70e7c31d620fbfb8ff4fa7f76861
Signed-off-by: Shivani Raval <sraval@habana.ai>
---

diff --git a/fs/jgs/rma_if_dev.h b/fs/jgs/rma_if_dev.h
new file mode 100644
index 0000000..8d8ac7e
--- /dev/null
+++ b/fs/jgs/rma_if_dev.h
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: MIT
+ *
+ * Copyright 2025 Intel Ltd.
+ * All Rights Reserved.
+ */
+
+#ifndef _JGS_RMA_IF_DEV_H_
+#define _JGS_RMA_IF_DEV_H_
+
+/* TODO: set proper values for JGS */
+#define MAX_RMA_HEADER_ENTRIES			1
+#define MAX_SRAM_ECC_ENTRIES			1
+#define MAX_HBM_ECC_ENTRIES			1
+#define MAX_PARITY_ENTRIES			1
+#define MAX_ROW_REPLACE_ENTRIES			1
+#define MAX_PAGE_DISCARDS_ENTRIES		1
+#define MAX_HBM_LANE_REPAIR_ENTRIES		1
+#define MAX_HBM_SPARE_ROWS_ENTRIES		1
+
+#endif /* _JGS_RMA_IF_DEV_H_ */
